docs/api/dt/cummax.rst

```diff
@@ -15,6 +15,11 @@
     cols: FExpr
         Input data for cumulative maximum calculation.
 
+    reverse: bool
+        If ``False``, the cumulative maximum is computed from the last row
+        to the first row. if ``True``, the cumulative maximum is computed 
+        from the first row to the last row.
```

Comment:
I guess it is vice versa: 
- `False` -> from the first to the last row;
- `True` -> from the last to the first row.

Also, `reverse` parameter is missing in the function signature.